app/static/src/app/components/DownloadOutput.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div>
+    <div v-if="open" class="modal-backdrop fade show"></div>
+    <div class="modal" :class="{show: open}" :style="style">
+      <div class="modal-dialog modal-dialog-centered">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">Download Run</h5>
+          </div>
+          <div class="modal-body">
+            <div class="row" id="download-file-name">
+              <div class="col-4">
+                <label class="col-form-label">File name</label>
+              </div>
+              <div class="col-8">
+                <input type="text" class="form-control" :value="fileName" @change="updateUserFileName">
+              </div>
+            </div>
+            <div class="row mt-2" id="download-points">
+              <div class="col-4">
+                <label class="col-form-label">Modelled points</label>
+              </div>
+              <div class="col-8">
+                <input type="number" class="form-control" v-model="points">
```

Comment:
Validation will be need for this input field.

Reply:
We should keep it above 0, and yeah, I think it would make sense to have some sort of maximum!

Reply:
I tried this with 50000 points or so and it was fine, really. Produced a 10MB file but it was fast enough. This only affects the user's browser, so if they want to go overboard that feels ok really